src/modules/monitoring/plugin.js

```diff
@@ -0,0 +1,49 @@
+const cron = require('node-cron');
```

Comment:
Presumably this plugin is not needed any more?

It does have some issues as it runs a job on a cron without using the job queue, so it will run on each Node process simultaneously (e.g. 4x )